### DISCUSSION TITLE
Can mplhep use in swan(cern)?

### DISCUSSION THREAD

**Discussion by ZhenxuanZhang-Jensen** [scikit-hep/mplhep#384] [Q&A] [answered]
Hi, 
   I basically use SWAN to do my analysis work, and I know mplhep is really good at plotting, but I can't find any SWAN LCG release that can import mplhep, so if you know anything about this, please help me. thanks in advance!
best

**Comment by andrzejnovak** [comment, marked as answer]
@ZhenxuanZhang-Jensen relatively recent version should be pulled in with `coffea` in LCG, otherwise you should be able to run `!pip install mplhep` in the notebook to install latest version from pypi on top of the LCG stack

**Reply by ZhenxuanZhang-Jensen** [reply]
okay, the versions with `coffea` and `mplhep` are not supported in SWAN yet, but pip works fine with me. thanks a lot!
